src/components/Documents/CodeOfConduct.tsx

```diff
@@ -0,0 +1,224 @@
+			consistently breaking a social rule, ask an organizer or moderator for help.
+		</p>
+		<p>
+			If you have any questions about any part of the code of conduct or social rules, please feel free to reach out to any of organizers or moderators.
+		</p>
+
+		<h3>Contact Information</h3>
+		<p>
+			If you have any concerns around social conduct or notice that someone may be the target harassment or oppressive behaviour, please reach out to a Toronto JS organizer
+			or volunteer. Please contact
+			<a href='https://torontojs.com/p/volunteers'>a member of Toronto JS organizers or volunteer staff</a>. You can use the below form to contact us, with or without your
+			name.
+		</p>
+		<p>
+			Organizers, moderators and volunteers will be happy to do whatever we can to help those experiencing harassment to feel safe during events and at all times in our
+			online community. We value your attendance.
+		</p>
+
+		<div className='hide-print'>
+			<h3>
+				Report a Violation (with or without your name)
+			</h3>
+			<p>Please provide the following information:</p>
+			<ul>
+				<li>the time and place the harassment took place</li>
+				<li>the member(s) involved in the event</li>
+				<li>any surrounding details you think are relevant to the report</li>
+			</ul>
+			<form
+				action='https://formspree.io/f/mayvyvzk'
+				method='POST'
+			>
+				<label htmlFor='report-replyto'>Email</label>
+				<input type='email' name='_replyto' id='report-replyto' />
+				<br />
+				<label htmlFor='report-message'>Description</label>
+				<textarea id='report-message' name='message' rows={5}></textarea>
+				<br />
+				<button type='submit'>Send</button>
+			</form>
+		</div>
+
+		<h3>License</h3>
+		<p>
+			The Toronto JS Code of Conduct is under a <a href='http://creativecommons.org/about/cc0'>Creative Commons Zero</a> license. Our Code of Conduct was adapted from the
+			{' '}
+			<a href='https://dev.to/code-of-conduct'>Dev.to Code Of Conduct.</a> It was forked from the{' '}
+			<a href='http://bangbangcon.com/conduct.html'>!!Con Code of Conduct</a>, which is also licensed under a{' '}
+			<a href='http://creativecommons.org/about/cc0'>Creative Commons Zero</a> license, and which itself was forked from{' '}
+			<a href='https://us.pycon.org/2013/about/code-of-conduct/'>PyCon 2013 Code of Conduct</a>, which is licensed under a{' '}
+			<a href='http://creativecommons.org/licenses/by/3.0/'>Creative Commons Attribution 3.0 Unported License</a>, and which itself was forked from{' '}
+			<a href='http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy'>
+				an example policy from the Geek Feminism wiki, created by the Ada Initiative and other volunteers
+			</a>{' '}
+			and available under a <a href='http://creativecommons.org/about/cc0'>Creative Commons Zero</a> license.
+		</p>
+	</>
+);
```

Comment:
Nitpick:

Left, right and bottom spacing for all tree disclosure docs to be pleasant to read and professional. 

![image](https://github.com/user-attachments/assets/d8a06ba8-9004-4a84-b0d4-e24b75bb2e06)